docs/reference-guides/block-api/block-registration.md

```diff
@@ -227,6 +227,26 @@ example: {
 },
 ```
 
+```js
+example: {
+	attributes: {
+		cover: 'https://example.com/image.jpg',
+	},
+	scale: 0.8
```

Comment:
Can't we achieve the same result with the existing `viewportWidth` somehow?

Reply:
It can be achieved with `viewportWidth`, but numbers are confusing. Preview panel is `150px`, however to fit the block properly, it requires numbers such as `400` or `500` which is very confusing.
The `scale` is much simpler which defaults to `1` meaning actual size and `0.8` means shrink to `80%` of the actual size.

Reply:
>it requires numbers such as 400 or 500 which is very confusing.

I think that's subjective and `viewportWidth` is used for a while now. 🤷 It seems we try to accommodate for the specific height of this preview that overflows and the others don't?

Reply:
Yeah I'm mixed on this. On the one hand `scale` is arguably a nicer API, but on the other hand you can already accomplish this with `viewportWidth` which exists already. We have to be really careful when adding new APIs to `block.json` as once they're in Core we basically have to maintain them forever. (Core has a very strict backwards compatibility policy.)

Reply:
Yeah, let's try to leverage what exists. If we need to allow for scale outside of viewportWidth the name should also be viewportScale.

Reply:
Thanks for the wonderful feedback. Reverted the new APIs and making use of `viewportWidth`